packages/ckeditor5-editor-classic/src/classiceditor.js

```diff
@@ -77,8 +77,9 @@ export default class ClassicEditor extends Editor {
 		this.model.document.createRoot();
 
 		const shouldToolbarGroupWhenFull = !this.config.get( 'toolbar.shouldNotGroupWhenFull' );
+		const editorName = this.name;
 		const view = new ClassicEditorUIView( this.locale, this.editing.view, {
-			shouldToolbarGroupWhenFull
+			shouldToolbarGroupWhenFull, editorName
```

Comment:
Tests missing for `ClassicEditor` that would verify that `editorName` is passed to its view.